packages/js/product-editor/src/components/iframe-editor/keyboard-shortcuts/keyboard-shortcuts.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import { useDispatch, useSelect } from '@wordpress/data';
+import { useContext } from '@wordpress/element';
+import { useShortcut } from '@wordpress/keyboard-shortcuts';
+import {
+	store as interfaceStore,
+	// @ts-expect-error No types for this exist yet.
+} from '@wordpress/interface';
+
+/**
+ * Internal dependencies
+ */
+import { EditorContext } from '../context';
+import {
+	SETTINGS_SIDEBAR_IDENTIFIER,
+	SIDEBAR_COMPLEMENTARY_AREA_SCOPE,
+} from '../constants';
+
+export const KeyboardShortcuts = () => {
```

Comment:
Can we transform this into a hook? Since it's not rendering anything and is only executing a side effect, it would make more sense to handle this code as a hook rather than as a normal component.

Reply:
I thought the same thing. But, this needs to be under the `<BlockEditorProvider>`, so that it is in the right context. I could create a wrapper component that used the hook under there, but that wrapper component would also render nothing other than its children.

This is how Gutenberg handles this in its editors.